W-18359576 dw pdk gr (#874)
mulesoft/docs-gateway: pdk/1.4/modules/ROOT/pages/policies-pdk-create-schema-definition.adoc

   vars: []
----

> * `payload`: The body of the incoming request. Define the `mimeType` as either `text`, `json`, or `xml` to set how to read the payload. By default, `mimeType` is set to `text`. If the `content-type` header are bound to the evaluator then the payload is interpreted according to the value of the `content-type` header. To restrict the use of the `payload` variable, you must exclude the payload variable, for example:

Reviewer: ```suggestion
* `payload`: The body of the incoming request. Define the `mimeType` as either `text`, `json`, or `xml` to set how to read the payload. By default, `mimeType` is set to `text`. If the `content-type` header is bound to the evaluator, then the payload is interpreted according to the value of the `content-type` header. To restrict the use of the `payload` variable, exclude the payload variable as shown in this example:
```
